### PR TITLE
Update scenarios label

### PR DESCRIPTION
**Pull request by AlteredCoder**
- Update scenario label (mostly capitalize)

### CODE REVIEW

File: scenarios/crowdsecurity/http-bf-wordpress_bf.yaml
Comment (LaurenceJJones) [inline scenarios/crowdsecurity/http-bf-wordpress_bf.yaml:1]:
```
description: "Detect WordPress Bruteforce"
```

File: scenarios/crowdsecurity/http-crawl-non_statics.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 type: leaky
 name: crowdsecurity/http-crawl-non_statics
-description: "Detect aggressive crawl from single ip"
+description: "Detect aggressive crawl on non static ressources"
```
Comment (LaurenceJJones) [inline scenarios/crowdsecurity/http-crawl-non_statics.yaml:3]: `resources` <-- spell check

File: scenarios/crowdsecurity/http-bf-wordpress_bf.yaml
Comment (AlteredCoder) [reply scenarios/crowdsecurity/http-bf-wordpress_bf.yaml:1]: nice catch!

**Review by blotus — APPROVED** [review]

**Review by LaurenceJJones — APPROVED** [review]
LGTM

**Pull request merged**